rules/core/media-feature-colon-space-after/README.md

```diff
@@ -0,0 +1,27 @@
+#### 📍 media-feature-colon-space-after
+
+Require a single space after the colon in media features.
+
+##### 🧟 Example of incorrect code for this rule:
```

Comment:
Please change to new file structure. see https://github.com/netsells/stylelint-config-netsells/pull/17 for an example